### DISCUSSION TITLE
Linux Compatibility

### DISCUSSION THREAD

**Discussion by shorent** [jwallet/spy-spotify#455] [Q&A] [answered]
Does this software work on natively linux? Are there any plans to get this working natively on linux?

**Comment by jwallet** [comment, marked as answer]
this is only available on Windows. Maybe it runs well in a Windows VM on Linux

**Reply by goguetchapuisb** [reply]
Did you think it's possible to code it for debian easily ?

**Reply by shorent** [reply]
Unfortunately, it would probably require a complete refactor of the projects code

**Comment by goguetchapuisb** [comment]
Somebody has tried ?
